### PR TITLE
chore(docs): broken markdown in rfcs

### PR DESCRIPTION
**Pull request by MarkMcCulloh**
There were two issue in the RFC markdown from https://github.com/winglang/wing/pull/1133 that caused the docsite to fail its build. I verified the fixes [here](https://github.com/winglang/docsite/pull/220).

Also made some tweaks that should make it a little prettier. Below is what it looked like before because they didn't have proper title/descriptions

image

*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/05-rfcs/process.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+title: RFC Process
+description: How to create a ship a "Request For Comment"
```
Comment (Chriscbr) [inline docs/05-rfcs/process.md:3]:
```suggestion
description: How to create and ship a "Request For Comments"
```

**Review by Chriscbr — APPROVED** [review]
LGTM

**Comment by MarkMcCulloh** [comment]
@Chriscbr just FYI, automerge shouldn't happen unless all conversations are resolved

**Comment by Chriscbr** [comment]
@MarkMcCulloh That's convenient - thanks for the heads up

**Pull request merged**